### ISSUE TITLE
Disable SSL protocol in TLS_Server and TLS_Client profiles

### ISSUE THREAD

**Issue by oakh1** [F5Networks/f5-appsvcs-extension#392]
### Is your feature request related to a problem? Please describe.
When declaring a policy using a client or server tls profile, it seems like it's only possible to disable specific TLS protocols (1.0, 1.1 or 1.2). I can't seem to find a way to disable old SSL protocols? (SSLv3 etc)

### Describe the solution you'd like
An option to disable SSL protocols just as it's possible to disable specific TLS protocols.

### Additional context
![Screenshot from 2020-12-04 00-08-03](https://user-images.githubusercontent.com/64788752/101099406-dea2b780-35c4-11eb-89fb-f5ea884564c9.png)
![Screenshot from 2020-12-04 00-09-38](https://user-images.githubusercontent.com/64788752/101099499-085bde80-35c5-11eb-95ae-57c78a093358.png)

**Comment by mdditt2000** [comment]
@oakh1 you can use TLS options in AS3 to disable specific tls versions

https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/declarations/tls-encryption.html#configuring-additional-tls-options-on-a-virtual

Example:

```
"tls1_3Enabled": false,
"tls1_2Enabled": false,
"tls1_1Enabled": true,
"tls1_0Enabled": false,
```

**Comment by oakh1** [comment]
Didn't see that you had replied, sorry about that.

I know that you can disable TLS protocols in as3, and I wrote that as well. What I was asking if how you disable the **SSL** protocol (in other words, the predecessor to TLS) as a protocol such as SSLv3 is extremely insecure.

I suspect that it might be possible by using cipher-groups etc, but it would be good if you can just disable all of the SSL protocols just as you are in the GUI (see my picture). Or maybe there's another way to do it?

**Comment by dstokesf5** [comment]
Thank you for your feedback. This feature request has been added to our internal product backlog as AUTOTOOL-2730.

**Comment by dstokesf5** [comment]
This has been resolved in the 3.33.0 release.
